### PR TITLE
Newsfeed

### PR DESCRIPTION
**Pull request by kristwob**
start of newsfeed, added class Status and a template for the newsfeed along with minor updates to database class and functions

### CODE REVIEW

**Review by daniepbj — APPROVED** [review]
Looks good

**Comment by OlaHulleberg** [comment]
Looks good to me. One concern: Do we want to INSERT the status into the database when creating a new class instance of it though? That would prevent us from using the Status class for displaying the different statuses later.

Share your thoughts @kristwob @daniepbj

**Comment by kristwob** [comment]
I changed it later, just haven't pushed the changes yet

File: classes/class_Status.php [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
-?>
\ No newline at end of file
+            foreach ($friendPost as $post) {
+                $displayArray[$idx]['Username'] = $post['user_id'];
+                $displayArray[$idx]['Time'] = date('d/m/Y H:i', $post['status_time']);
+                $displayArray[$idx]['Content'] = $post['status_content'];
+                $idx++;
+            }
+        }
+        return $displayArray;
```
Comment (OlaHulleberg) [inline classes/class_Status.php:87]: I still feel like there should be an instance of the Status class with the values Username, Time and Content. But this approach works! :)

**Review by OlaHulleberg — APPROVED** [review]

**Pull request merged**